### DISCUSSION TITLE
ProfilingPlugin

### DISCUSSION THREAD

**Discussion by jianxingyao** [webpack/webpack#18962] [Q&A]
As per the official documentation, this is my use of the plugin
![image](https://github.com/user-attachments/assets/0e6dbf69-6e33-48bf-88b0-07329be1d1e7)
I got it
![image](https://github.com/user-attachments/assets/dbf705ef-f39f-4f6c-aa9e-71514200ff28)
Drag into the Performance tab and get it
![image](https://github.com/user-attachments/assets/dcc7f6a2-d447-48ca-80de-2c86d06c5246)

This result is too messy, according to the official documentation, there should only be an analysis of the use of plugins in the project, how can I streamline this result.

**Comment by alexander-akait** [comment]
This is the expected result. You can try - https://webpack.js.org/configuration/other-options/#profile, it is more readable for users

**Reply by jianxingyao** [reply]
I've tried this option, but the result is not what I wanted, it only seems to contain the build information of chunk, module, assets, I want to be able to visualize the time consumption of the plugin and loader, such as speed-measure-webpack-plugin, or whether the results of ProfilingPlugin can be optimized?

**Reply by alexander-akait** [reply]
You can use - new webpack.ProgressPlugin({ profile: true }), but it is also has a simple output, ProfilingPlugin made specifically for profiling, you can see literally everything there, if you need a simple output then use https://github.com/stephencookdev/speed-measure-webpack-plugin
